docs/src/tutorial-loop-unwinding.md

```diff
@@ -79,5 +79,39 @@ Failed Checks: dereference failure: pointer outside object bounds
 
 Kani is now sure we've unwound the loop enough to verify our proof harness, and now we're seeing just the bound checking failures from the off-by-one error.
 
+## Unwinding value specification
+
+Kani allows three options to specify the unwind value for a particular harness:
+
+1. The unwind annotation `#[kani::unwind(<num>)]`. This sets the unwind value for the harness with the annotation. Example:
+``` rust
+#[kani::proof]
+#[kani::unwind(3)]
+fn proof_harness() {
+[...]
+}
+```
+2. `--unwind` flag. This sets the unwind value for the entire file/crate on which kani or cargo-kani is called. Example -
+```
+kani file.rs --unwind 3
+```
+3. `--harness-unwind` flag. This needs to be used alongside `--harness` and sets the unwind value for the harness specified. Example -
```

Comment:
I'd insert a sentence:

```
3. `--harness-unwind` flag.  This overrides any annotation and forces the harness to use the specified value. ...
```

Then just remove the whole section on "order of precedence", since "default" and "override" should be clear enough.